CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# In a CODEOWNERS file, the usernames must begin with @
+
+# Global owners for entire repo:
+* @SaschaMann @IanButterworth @DilumAluthge
```

Comment:
hmmm I don't love codeowners files.. obligation takes the fun out of OSS

Go ahead without me though

Reply:
That's a fair point - I don't want people feeling obligated or pressured here.

I'm not sure if the benefits outweigh the costs here. Let's punt for now.